### ISSUE TITLE
DataprocPrestoJob operator

### ISSUE THREAD

**Issue by mik-laj** [PolideaInternal/airflow#789]
Hello,

We miss presto on Dataproc operator.
https://github.com/mik-laj/discovery-api/commit/efa369013feb249c4f37526e87b325273a66cc15

Best regards,
Kamil

**Comment by turbaszek** [comment]
New operators are job-type-independent. As per python client we have `DataprocSubmitJobOperator` which uses the following method https://googleapis.dev/python/dataproc/latest/gapic/v1beta2/api.html#google.cloud.dataproc_v1beta2.JobControllerClient.submit_job

**Comment by turbaszek** [comment]
Same goes for SparkRJob and

**Comment by mik-laj** [comment]
We should make these decisions together with the Dataproc team, because it is a key service. They may have a different vision of developing this integration.

**Comment by turbaszek** [comment]
@mik-laj composer team has a guide how to develop GCP operators. Why should we make an exception for Dataproc? I

**Comment by mik-laj** [comment]
Yes. If there is a business need, we should prepare additional operators that will facilitate the use of the service. The guide does not say that you cannot create additional operators. They describe basic cases, but there may be a need to create additional operators that meet the specific needs of users. We will have several operators that do the same, but if users expect it, we have to help them.

**Comment by mik-laj** [comment]
Operators must be useful to users, and they will not be if we follow strict rules. We already have several operators that are not covered by the guide. 
https://airflow.readthedocs.io/en/latest/_api/airflow/providers/google/cloud/operators/translate_speech/index.html#module-airflow.providers.google.cloud.operators.translate_speech
https://airflow.readthedocs.io/en/latest/_api/airflow/providers/google/cloud/operators/cloud_memorystore/index.html#airflow.providers.google.cloud.operators.cloud_memorystore.CloudMemorystoreExportAndDeleteInstanceOperator

**Comment by turbaszek** [comment]
Let's ask them
